skimage/registration/_phase_cross_correlation.py

```diff
@@ -172,11 +172,12 @@ def _disambiguate_shift(reference_image, moving_image, shift):
             max_slice, positive_shift, negative_shift
             ):
         real_shift_acc.append(pos_shift if sl.stop is None else neg_shift)
-    if not subpixel:
-        real_shift = tuple(map(int, real_shift_acc))
```

Comment:
And since `real_shift` was supposed to be a 'tuple of float,' it's not clear why this mapping to integer was even performed...

Reply:
I am also not seeing an immediate reason in https://github.com/scikit-image/scikit-image/pull/6617 why this mapping to int is performed. @jni do you remember?